config/lib/app.js

```diff
@@ -32,8 +26,14 @@ module.exports.init = function init(callback) {
 module.exports.start = function start(callback) {
   var _this = this;
 
+  _this.loadModels();
   _this.init(function (app, db, config) {
 
+    //Seed DB with initial values
+    if (config.seedDB) {
```

Comment:
does this seed the db every time we hot reload the server?

Reply:
Honestly not sure, good question. The seed file should be idempotent, so there shouldn't be any correctness problems.

I don't think we should expect performance issues either during development, because writing seed data to the db (or attempting to) should just be in the order of milliseconds.